debian/marquee/nginx/conf/whatwg.conf

```diff
@@ -0,0 +1,29 @@
+ssl_session_cache shared:SSL:10m;
```

Comment:
> Aside: here I was wondering if all of this could be hoisted to the http context to avoid the includes in each file, but it can't because http://nginx.org/en/docs/http/ngx_http_core_module.html#location wouldn't work then.

Right, yeah, that’s entirely the reason why I ended up needing to put them in this separate `whatwg.conf` file